pkg/admin/prerun/prerun.go

```diff
@@ -21,29 +21,41 @@ func NewDataManager(dataManager datadir.Manager) *DataManager {
 }
 
 func (dm *DataManager) Run() error {
-	klog.InfoS("Starting pre-run")
-	defer klog.InfoS("Pre-run complete")
-
```

Comment:
I think we would benefit from a message here and deferred closing scope. 
Otherwise, we may see non-ostree-based, etc. out of context.